packages/caravan-clients/src/bitcoind.js

```diff
@@ -144,16 +144,14 @@ export function bitcoindImportMulti({ url, auth, addresses, label, rescan }) {
   return callBitcoind(...params);
 }
 
-export async function bitcoindRawTxData(txid){
-  try{
-    return await callBitcoind(
-      this.bitcoindParams.url,
-      this.bitcoindParams.auth,
-      "decoderawtransaction",
-      [txid],
-    );
-  }
-  catch(e){
-    return e;
+export async function bitcoindRawTxData(url, auth, txid) {
+  try {
+    const response = await callBitcoind(url, auth, "getrawtransaction", [
+      txid,
+      true,
+    ]);
+    return response.result;
```

Comment:
when i run this with the #168 PR I'm getting an error because callBitcoind doesn't return a result prop. if you change this, the error doesn't happen anymore.